.gitignore

```diff
@@ -8,3 +8,6 @@ coverage
 /data
 /.env
 /build
+privacy.html
+
+meet-the-team.html
```

Comment:
Great job. However, I think you should specify the path to the html files in case another file with the same name exists in another folder. Also, there should be a new line at the end of this file.